src/schema/suffixes/bold.yaml

```
@@ -0,0 +1,4 @@
+---
+name: Blood-Oxygen-Level Dependent
```

I think you've been using image

```suggestion
name: Blood-Oxygen-Level Dependent image
```